### DISCUSSION TITLE
Support for aarch64 platform docker image

### DISCUSSION THREAD

**Discussion by include** [apache/superset#20735] [Ideas]
Not sure if there is someone working with it, but I would love to run `docker-compose -f docker-compose-non-dev.yml up` on my Raspberry Pi aarch64 :)

**Comment by srinify** [comment]
@betodealmeida has messed around with some raspberry pi + Superset stuff!

Moving to discussion as this isn't a bug :)

**Comment by redwards101** [comment]
I'm working with aarch64 cloud instances and it would be great to do the same.  Any plan to support it?

**Reply by AmitArie** [reply]
any update? aarch64 would be lovely!

**Comment by redwards101** [comment]
I haven't tried it lately since I just ended up using an amd64 image for it.  I really don't like haven't to do that, however.

This guy appears to have had success:
https://github.com/apache/superset/discussions/24071
